gnat2goto/driver/goto_utils.adb

```diff
@@ -329,6 +329,21 @@ package body GOTO_Utils is
                           I_Type          => Pointer_Type);
    end Offset_Array_Data;
 
+   function Float_Size (Entity_Esize : Uint) return Integer
+   is
+      Unknown_Float_Width : exception;
+   begin
+      if Entity_Esize = 32 then
+         --  23-bit mantissa, 8-bit exponent
+         return 23;
+      end if;
+      if Entity_Esize = 64 then
+         --  52-bit mantissa, 11-bit exponent
+         return 52;
+      end if;
+      raise Unknown_Float_Width;
```

Comment:
We should have a proper human readable error in case this ever happens in practice (which it well might)